tests/tests.py

```diff
@@ -519,3 +519,62 @@ def test_algerian_phone_number_in_model(self):
         )
         self.assertEqual(phonenumbers.parse(ALGERIAN_PHONE_NUMBER), m.phone_number)
         self.assertEqual(ALGERIAN_PHONE_NUMBER, m.phone_number)
+
+    # @override_settings(
+    #     PHONENUMBER_DEFAULT_REGION="AR",
+    #     PHONENUMBER_DEFAULT_FORMAT="NATIONAL"
+    # )
+    def test_international_phone_is_valid(self):
+        class PhoneNumberForm(forms.ModelForm):
+            class Meta:
+                model = models.TestModel
+                fields = ["phone"]
+
+        form = PhoneNumberForm({"phone": "+32468547825"})  # belgian phone
+        self.assertTrue(form.is_valid())
+        self.assertEqual(
+            form.as_p().replace('\n    ', '').replace('  \n  ', '').replace('label> ', 'label>'),
+            '<p><label for="id_phone">Phone:</label>'
+            '<input type="tel" name="phone" value="+32468547825" maxlength="128" required id="id_phone">'
+            '</p>'
+        )
+        form.save()
```

Comment:
Why save the form?

Reply:
Saving the form triggered cleaning twice which led me to a bug in the implementation.